### ISSUE TITLE
[exp-signin-rework] getRecomAutoplay breaking entire page

### ISSUE THREAD

**Issue by aubymori** [Rehike/Rehike#25]
```
Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in C:\Users\Aubrey\src\exp-signin-rework\models\Watch\Watch8\Watch8Subcontroller.php:230
Stack trace:
#0 C:\Users\Aubrey\src\exp-signin-rework\models\Watch\Watch8\Watch8Subcontroller.php(230): count(NULL)
#1 C:\Users\Aubrey\src\exp-signin-rework\models\Watch\Watch8\Watch8Subcontroller.php(104): Rehike\Model\Watch\Watch8\Watch8Subcontroller::getRecomAutoplay(NULL)
#2 C:\Users\Aubrey\src\exp-signin-rework\models\Watch\WatchModel.php(189): Rehike\Model\Watch\Watch8\Watch8Subcontroller::bakeSecondaryResults(Object(stdClass))
#3 C:\Users\Aubrey\src\exp-signin-rework\models\Watch\WatchModel.php(86): Rehike\Model\Watch\WatchModel::bakeSecondaryResults(Object(stdClass))
#4 C:\Users\Aubrey\src\exp-signin-rework\controllers\watch.php(138): Rehike\Model\Watch\WatchModel::bake(Object(stdClass), Object(stdClass), Object(stdClass))
#5 C:\Users\Aubrey\src\exp-signin-rework\controllers\core\HitchhikerController.php(27): Rehike\Controller\core\NirvanaController@anonymous->onGet(Object(stdClass), Object(Rehike\ControllerV2\RequestMetadata))
#6 C:\Users\Aubrey\src\exp-signin-rework\modules\Rehike\ControllerV2\GetControllerInstance.php(51): Rehike\Controller\core\HitchhikerController->get(Object(stdClass), '', Object(Rehike\ControllerV2\RequestMetadata))
#7 C:\Users\Aubrey\src\exp-signin-rework\modules\Rehike\ControllerV2\GetControllerInstance.php(73): Rehike\ControllerV2\GetControllerInstance->_cv2WrappedCallControllerMethod('get', Array)
#8 C:\Users\Aubrey\src\exp-signin-rework\modules\Rehike\ControllerV2\Router.php(204): Rehike\ControllerV2\GetControllerInstance->__call('get', Array)
#9 C:\Users\Aubrey\src\exp-signin-rework\modules\Rehike\ControllerV2\Router.php(148): Rehike\ControllerV2\Router::pointerHandler('watch', 'get')
#10 C:\Users\Aubrey\src\exp-signin-rework\modules\Rehike\ControllerV2\Router.php(25): Rehike\ControllerV2\Router::baseRequestMethod(Array, 'get')
#11 C:\Users\Aubrey\src\exp-signin-rework\router_v2.php(25): Rehike\ControllerV2\Router::get(Array)
#12 C:\Users\Aubrey\src\exp-signin-rework\index.php(45): require('C:\\Users\\Aubrey...')
#13 {main}
  thrown
  ```

**Comment by YukisCoffee** [comment]
Not enough details to replicate this. Could you elaborate on how you got this error?

**Comment by aubymori** [comment]
@YukisCoffee Specific videos do this for no reason and I have no idea why. I don't have one on hand, but if I find one I'll notify

**Comment by YukisCoffee** [comment]
I believe this was fixed by 709ab8e14017477631b8e4fb3ea4352ce0bcf1dc on `exp-signin-rework`.

I'll close this, reopen if it still is not working.
